src/extension.js

```diff
@@ -1,49 +1,52 @@
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
-const vscode = require( "vscode" );
-const BoxCommand = require( "./box-command" );
+const vscode = require("vscode");
+const BoxCommand = require("./box-command");
+const { LOG } = require('./utils/logger');
 
-let globalCommand;
+let globalCommand = new BoxCommand({ runHarness: true });
```

Comment:
Adding the global command since it gets set later, but initially it throws an error